### DISCUSSION TITLE
Source Terms Units ?

### DISCUSSION THREAD

**Discussion by AnalyzeFTW** [nextfoam/baram#47] [Q&A]
Hello there.  Under Cell Zone Conditions what are the units for Energy and Mass ? (eg TKE source units are mentioned but not for energy and mass sources) .Thank you

**Comment by jiban** [comment]
Hi, it's **_W_**(watt) for energy and **_kg/s_** for Mass.
I think it should be displayed.
Thank you.

**Comment by kdg5424** [comment]
Hello, AnalyzeFTW. In Energy case, if you choose **unit per volume** unit will be W/m^3, and if you choose **entire cell** then unit will be W.
Thank you.
